keyboards/jadookb/jkb65/JKB65.c

```diff
@@ -0,0 +1,84 @@
+#include "jkb65.h"
```

Comment:
These two files still need to be lowercase.

Reply:
done